Makefile

```diff
@@ -164,3 +164,6 @@ python-build:
 python-test: 
 	$(MAKE) python-build extras="[tests]"
 	DEBUG=1 ${PYTHON_EXECUTABLE} -m unittest -v
+
+cmake-gpu:
```

Comment:
nit: why not just name it gpu (any reason to prefix cmake ?)

Reply:
Because it only generate the rules to do the build but not actually building.  It is functionally equivalent to the `cmake` target above.